example/logrus/main.go

```diff
@@ -43,9 +43,16 @@ func main() {
 	defer sentryHook.Flush(5 * time.Second)
 	logger.AddHook(sentryHook)
 
+	// Flushes before calling os.Exit(1) when using logger.Fatal
+	// (else all defers are not called, and Sentry does not have time to send the event)
+	logrus.RegisterExitHandler(func() { sentryHook.Flush(5 * time.Second) })
```

Comment:
Would it not make sense to do this unconditionally, so the caller doesn't need to?

Reply:
We could, but it is unnecessary if Sentry sends its events synchronously.